### ISSUE TITLE
Docker installation not working

### ISSUE THREAD

**Issue by pezzeonline** [fonoster/fonoster#430]
**Describe the bug**
When I launch the command in the documentation I receive always the same error:

`ERROR: for mediaserver  Container "327cdb9899a6" is unhealthy.`

`docker logs` give me this error on the mediaserver container:

```
[INFO  tini (1)] Spawned child process './routr' with pid '8'
[INFO  tini (1)] Main child exited with signal (with signal 'Terminated')
```

**To Reproduce**
1. Run the suggested command:
```
docker run -it --rm \
  -e CONFIG_PATH=$(pwd)/fonoster/config \
  --volume /var/run/docker.sock:/var/run/docker.sock \
  --volume $(pwd)/fonoster:/out:rw \
  fonoster/fonoster
```
2. Wait for the error

**Expected behavior**
Application has to start

**System information (please complete the following):**
 - OS: Ubuntu Server 22.04 on Intel, Mac OSX 12.6 on Apple M1

**Comment by psanders** [comment]
Hi @pezzeonline,
 
Thanks for taking the time to try Fonoster. We will have someone on the team triaging this issue.

cc: @YuriCodes @efraa

**Comment by YuriCodes** [comment]
Hello @pezzeonline thank you for reaching out. We are currently working on giving you an accurate solution so you can get back to your project. Thank you for your patience

**Comment by psanders** [comment]
@efraa I am bringing this to the board as the next thing that needs attention.

**Comment by chknus** [comment]
Any updates on this? 😊

**Comment by psanders** [comment]
@chknus we can confirm we are seeing the same issue. 

However, we looking at the root of the problem.

I hope to have more information next week.

**Comment by pezzeonline** [comment]
By the way if you start mediaserver alone it is working well using documented command:

```
docker run -it \
    -p 6060:6060 \
    -p 8088:8088 \
    -p 10000-10010:10000-10010 \
    -e EXTERN_ADDR=${you host address} \
    -e SIPPROXY_HOST=${sip proxy address} \
    -e SIPPROXY_USERNAME=${username at sip proxy} \
    -e SIPPROXY_SECRET=${secret at sip proxy} \
    -e RTP_PORT_START=10000 \
    -e RTP_PORT_END=10010 \
    -e ENABLE_TEST_ACCOUNT=true \
    fonoster/mediaserver
```

Same information (not error as I wrote): 
`[INFO  tini (1)] Spawned child process '/run.sh' with pid '8'`
but boot finish without problems.

**Comment by psanders** [comment]
When running as part of Fonoster, the MediaServer depends on Routr.

For some reason, the MediaServer can't register to Routr, showing an Unhealthy state.

We didn't make much progress on this issue, but I will try to keep this thread updated.

**Comment by psanders** [comment]
@efraa I was able to install Fonoster on my Intel, however it failed on the M1. I will do more tests, but it feels like M1s that might be the problem.

**Comment by kymppi** [comment]
It's probably something with the CPU architecture. Intel cpus have x86_64 but M1 chips use arm. Most likely you will need to add cross-building

**Comment by psanders** [comment]
@kymppi we checked an confirm is not an issue with the CPU architecture since Routr runs fine as a standalone service using Docker. We will continue triaging this issue and keep the thread updated.

cc @pezzeonline

**Comment by pezzeonline** [comment]
> @kymppi we checked an confirm is not an issue with the CPU architecture since Routr runs fine as a standalone service using Docker. We will continue triaging this issue and keep the thread updated.
> 
> cc @pezzeonline

Thank you. Actually, as I wrote in the first post, I have the same error on a Intel server with Ubuntu 22.04 LTS

**Comment by kymppi** [comment]
> Thank you. Actually, as I wrote in the first post, I have the same error on a Intel server with Ubuntu 22.04 LTS

Oh lmao, I just read the macos part :P

**Comment by psanders** [comment]
Quick update.

**TLDR: Made some progress but it seems that this installer needs a lot more work.**

Found few issues with the docker installer.

Addressed the issues in the [this branch](https://github.com/fonoster/fonoster/compare/main...fix/broken-docker-installer).

Tested the fixes on Ubuntu @ DigitalOcean and my M1.

After installing I had to stop/start the server due to the MediaServer.

The installation is considerably slower on my M1 (potentially due to QEMU.)

More to follow.
